app/background/events/shieldsEvents.ts

```diff
@@ -5,6 +5,11 @@
 import actions from '../actions/shieldsPanelActions'
 import { BlockDetails } from '../../types/actions/shieldsPanelActions'
 
-chrome.braveShields.onBlocked.addListener((detail: BlockDetails) => {
-  actions.resourceBlocked(detail)
-})
+if (chrome.braveShields) {
+
```

Comment:
Just a couple minor nits and we should be good to go:
- Remove the blank line on line 9 
- Use 2 spaces instead of a tab on line 10, 11, 12, and 14.

You can run `npm run lint` to make sure the lint works.


After you make the changes to your branch you can `git add -u` and then `git commit --amend`, then `git push -f` to update the PR.